plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/delete/PositionDeleteWriter.java

```diff
@@ -88,26 +83,7 @@ public IcebergPositionDeletePageSink(
         this.writer = fileWriterFactory.createPositionDeleteWriter(fileSystem, Location.of(outputPath), session, fileFormat, storageProperties);
     }
 
-    @Override
-    public long getCompletedBytes()
```

Comment:
It would be nice to track bytes written somewhere too. It will make the bytes written for query metrics more accurate, and it's also a factor in the heuristic for writer scaling I believe.
cc: @gaurav8297 @sopel39